gateway-service/src/main/java/org/zowe/apiml/gateway/service/RouteLocator.java

```diff
@@ -50,11 +50,17 @@ public class RouteLocator implements RouteDefinitionLocator {
     private boolean forwardingClientCertEnabled;
 
     @Value("${apiml.gateway.servicesToLimitRequestRate:-}")
+    List<String> servicesToLimitRequestRateProperty = Collections.emptyList();
     List<String> servicesToLimitRequestRate;
 
+    @Value("${apiml.gateway.servicesToDisableRetry:-}")
+    List<String> servicesToDisableRetryProperty = Collections.emptyList();
```

Comment:
Setting up `Collections.emptyList()` to properites is redundant in this case, because they always by default will be with one element - `-`. And to make them empty List by default it's better to remove `-`, like this:
```suggestion
    @Value("${apiml.gateway.servicesToLimitRequestRate:}")
    List<String> servicesToLimitRequestRateProperty;
    List<String> servicesToLimitRequestRate;

    @Value("${apiml.gateway.servicesToDisableRetry:}")
    List<String> servicesToDisableRetryProperty;
```

Reply:
Your statement is valid for spring boot initialized tests only, but cases NPE in unit tests.

Reply:
I moved the initialization to the test.

Reply:
Thanks!